examples/plot_connectivity.py

```diff
@@ -30,73 +33,99 @@
 # reflect the canonical neocortical microcircuit. ``net.connectivity``
 # is a list of dictionaries which detail every cell-cell, and drive-cell
 # connection.
-print(len(net.connectivity))
-print(net.connectivity[0:2])
+print(len(net_erp.connectivity))
+print(net_erp.connectivity[0:2])
 
 ###############################################################################
 # Data recorded during simulations are stored under
 # :class:`~hnn_core.Cell_Response`. To test multiple network structures, we can
 # create a copy of the original network. The copied network is then simulated.
-net_erp = net.copy()
 dpl_erp = simulate_dipole(net_erp, n_trials=1)
 net_erp.cell_response.plot_spikes_raster()
 
 ###############################################################################
-# We can modify the connectivity list to test the effect of different
-# connectivity patterns. For example, we can remove all layer 2 inhibitory
-# connections. In the default network, the src_gids of each connection are
-# all the same cell type.. Connections are stored under ``conn['gid_pairs']``
-# as a dictionary indexed by src_gid:
-# ``{src_gid1: [target_gid1, target_gid2], ...]``. Each src_gid indexes a
-# list with its target gids
-new_connectivity = [conn for conn in net.connectivity
-                    if conn['src_type'] != 'L2_basket']
-net.connectivity = new_connectivity
-
-net_remove = net.copy()
-dpl_remove = simulate_dipole(net_remove, n_trials=1)
-net_remove.cell_response.plot_spikes_raster()
-
-###############################################################################
-# That's a lot of spiking! Since basket cells are inhibitory, removing these
-# connections increases network wide excitability. We can additionally add
-# new connections using ``net.add_connection()``. Let's try connecting a
-# single layer 2 basket cell, to every layer 2 pyramidal cell. We can utilize
-# ``net.gid_ranges`` to help find the gids of interest.
+# We can also define our own connections to test the effect of different
+# connectivity patterns. To start, an empty network is instantiated.
+# ``add_drives_from_params`` can still be used with an empty Network, but
+# all cells will start disconnected. ``net.add_connection`` is then
+# used to create a custom network. Let us first create an all-to-all
+# connectivity pattern between the L5 pyramidal cells, and L2 basket cells.
 # :meth:`hnn_core.Network.add_connection` allows connections to be specified
 # with either cell names, or the gids directly. If multiple gids are provided
 # for either the sources or the targets, they will be connected in an
 # all-to-all pattern.
-print(net.gid_ranges)
-src_gid = net.gid_ranges['L2_basket'][0]
-target_gids = 'L2_pyramidal'
+
+net_all = Network(params, add_drives_from_params=True)
+
+# Pyramidal cell connections
+location, receptor = 'distal', 'ampa'
+weight, delay, lamtha = 1.0, 1.0, 70
+src = 'L5_pyramidal'
+for target in ['L5_pyramidal', 'L5_pyramidal']:
```

Comment:
pyramidal twice? why so?

Reply:
Definitely a mistake, fixing now. The goal is to just create connections within and between the different cells classes.